edunext_openedx_extensions/edunext/management/commands/import_xml_db.py

```diff
@@ -53,28 +72,34 @@ def handle(self, *args, **options):
                     as_range = range(int(limits[0]), int(limits[1]) + 1)
                     user_ids = user_ids + as_range
                 except Exception:
-                    raise CommandError("Could not process imput {}".format(statement))
+                    raise CommandError(
+                        "Could not process imput {}".format(statement))
 
-        logger.info("Users to import: {}".format(user_ids))
-        logger.info("Microsite: {}".format(microsite))
+        LOGGER.info("Users to import: {}".format(user_ids))
+        LOGGER.info("Microsite: {}".format(microsite))
 
-        # TODO: if we validate this, we might as well validate it for real with a dom parser or similar
+        # TODO: if we validate this, we might as well validate it for real with
+        # a dom parser or similar
         if not filename.endswith(".xml"):
-            raise CommandError("File {} is not an xml (*.xml)".format(filename))
+            raise CommandError(
+                "File {} is not an xml (*.xml)".format(filename))
 
-        importer = DatabaseXmlDumpImporter(filename, dry_run)
+        if FLAG_IMPORT:
```

Comment:
no need for this, let's leave it fail badly if the import was not done correctly